examples/custom_sensor/Odometer.cc

```diff
@@ -53,7 +53,7 @@ bool Odometer::Load(const sdf::Sensor &_sdf)
   }
 
   // Load custom sensor params
-  auto customElem = _sdf.Element()->GetElement("gz:odometer");
+  auto customElem = _sdf.Element()->FindElement("gz:odometer");
```

Comment:
you should check here if `customElem` is different to `nullptr`
```suggestion
  auto customElem = _sdf.Element()->FindElement("gz:odometer");
  if (customElem)
  {
```

same below

Reply:
actually there is a `HasElement("gz:odometer")` check above already so should be fine. Same for the one below.